### PR TITLE
[MXNET-1181] Added command line alternative to IntelliJ in install instructions

### PR DESCRIPTION
**Pull request by piyushghai**
## Description ##
Added Maven package setup instructions as an alternative to IntelliJ. Also removed a duplicate Java setup/install markdown file.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)

@lanking520 @andrewfayres

### CODE REVIEW

File: docs/tutorials/java/mxnet_java_on_intellij.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ TODO
 After clicking Finish, you will be presented with the project's first view.
 The project's `pom.xml` will be open for editing.
 
+**IntelliJ IDEA Alternative** If you want to use only Mave to create the project, you can create a new folder and run the following in the newly created folder :
```
Comment (kalyc) [inline docs/tutorials/java/mxnet_java_on_intellij.md]: [typo] Maven

File: docs/tutorials/java/mxnet_java_on_intellij.md [hunk 2]
```diff
@@ -108,6 +108,12 @@ TODO
...
+```bash
```
Comment (kalyc) [inline docs/tutorials/java/mxnet_java_on_intellij.md]: please add further clarification here:
mkdir example
cd example/
then run the command

Comment (piyushghai) [reply docs/tutorials/java/mxnet_java_on_intellij.md]: Thanks for pointing it out. Updated it.

File: docs/tutorials/java/mxnet_java_on_intellij.md (hunk 1 above)
Comment (piyushghai) [reply docs/tutorials/java/mxnet_java_on_intellij.md]: Thanks for pointing it out. Updated it.

File: docs/tutorials/java/mxnet_java_on_intellij.md [hunk 3]
```diff
@@ -108,6 +108,15 @@ TODO
 After clicking Finish, you will be presented with the project's first view.
 The project's `pom.xml` will be open for editing.
 
+**IntelliJ IDEA Alternative** If you want to use only Maven to create the project, you can create a new folder and run the following in the newly created folder :
+```bash
+mkdir java-proj
+cd java-proj
```
Comment (kalyc) [inline docs/tutorials/java/mxnet_java_on_intellij.md]: [minor] cd java-proj/

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-merge]

**Pull request merged**